### PR TITLE
Upgrade Decidim 0.27

### PR DESCRIPTION
**Pull request by antopalidi**

### CODE REVIEW

File: app/models/decidim/action_delegator/participant.rb [hunk 1]
```diff
@@ -21,7 +21,6 @@ class Participant < ApplicationRecord
       delegate :consultation, to: :setting
       delegate :organization, to: :setting
 
-      validates :setting, presence: true
```
Comment (microstudi) [inline app/models/decidim/action_delegator/participant.rb:24]: Why is this removed?

File: decidim-action_delegator.gemspec [hunk 2]
```diff
@@ -7,7 +7,7 @@ require "decidim/action_delegator/version"
 Gem::Specification.new do |s|
   s.version = Decidim::ActionDelegator::VERSION
   s.authors = ["Pau Pérez Fabregat", "Ivan Vergés"]
-  s.email = ["saulopefa@gmail.com", "ivan@pokecode.net"]
+  s.email = %w(saulopefa@gmail.com ivan@pokecode.net)
   s.license = "AGPL-3.0"
   s.homepage = "https://github.com/coopdevs/decidim-module-action_delegator"
   s.required_ruby_version = ">= 2.7"
```
Comment (microstudi) [inline decidim-action_delegator.gemspec:13]:
```suggestion
  s.required_ruby_version = ">= 3.0"
```

File: details.csv [hunk 3]
```diff
@@ -1 +0,0 @@
-,,,"Email can't be blank, Phone can't be blank"
```
Comment (microstudi) [inline details.csv:1]: Is not used in a test this?

File: lib/decidim/action_delegator/version.rb [hunk 4]
```diff
@@ -3,8 +3,8 @@
 module Decidim
   # This holds the decidim-action_delegator version.
   module ActionDelegator
-    VERSION = "0.7.2"
-    DECIDIM_VERSION = "0.26.5"
-    COMPAT_DECIDIM_VERSION = [">= 0.26.0", "< 0.27"].freeze
+    VERSION = "0.7.3"
```
Comment (microstudi) [inline lib/decidim/action_delegator/version.rb:6]:
```suggestion
    VERSION = "0.8.0"
```

File: spec/fixtures/details.csv [hunk 5]
```diff
@@ -1,3 +0,0 @@
-,,1.5,Phone can't be blank
-user@example.org,,1.5,Phone can't be blank
-user@example.org,6660008,member,Ponderation type not found
```
Comment (microstudi) [inline spec/fixtures/details.csv:3]: Why is this removed?

**Review by microstudi — CHANGES_REQUESTED** [review]
Thanks @antopalidi 
everything looks good a part from a couple of comments.
There's also one thing missing, update the README with the compabilble versions (see that the next version must be 0.8 not 0.7.3).

File: app/models/decidim/action_delegator/participant.rb (hunk 1 above)
Comment (antopalidi) [reply app/models/decidim/action_delegator/participant.rb:24]: belongs_to :setting already includes presence validation. So, the explicit validation is redundant.

Comment (microstudi) [reply app/models/decidim/action_delegator/participant.rb:24]: great, thanks
